.github/workflows/e2e.yml

```diff
@@ -160,19 +160,22 @@ jobs:
         working-directory: opencollective-frontend
         run: CYPRESS_INSTALL_BINARY=0 npm ci --prefer-offline --no-audit
 
+      - name: Set commit hash (frontend)
+        working-directory: opencollective-frontend
+        run: echo "FRONTEND_COMMIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+
       - name: Restore build (frontend)
         uses: actions/cache@v1
         id: next-build
         with:
           path: opencollective-frontend/dist
-          key: ${{ runner.os }}-next-build-${{ hashFiles('opencollective-frontend/**') }}
+          key: ${{ runner.os }}-next-build-${{ env.FRONTEND_COMMIT_HASH }}
 
       - name: Restore .next cache (frontend)
-        if: steps.next-build.outputs.cache-hit != 'true'
         uses: actions/cache@v1
         with:
-          path: opencollective-frontend/.next/cache
-          key: ${{ runner.os }}-next-cache-${{ hashFiles('opencollective-frontend/**') }}
+          path: opencollective-frontend/.next
```

Comment:
Why did you decide to cache the whole .next directory and not the cache directory like before?

Reply:
@znarf : Basically, if we don't cache the whole .next directory then when the e2e tests are run we get the following error due to the `.next` directory missing some things.

https://github.com/opencollective/opencollective-api/runs/3077008271

I've explained my rational for doing this at, https://github.com/opencollective/opencollective-api/pull/6274#discussion_r670552256. Let me know what you think on this. 🙂

Reply:
It was working before and for months. How do you explain that it would now suddenly failing?

Reply:
@znarf : Yes, it was working, but each time it was working, the build step was run. Note that we had the same condition `if: steps.next-build.outputs.cache-hit != 'true'` for both this and the build step. So essentially, after this `.next/cache` is restored the build step was run again. The build step replaces the whole `.next` folder. Thus I think previously there was no point of running this cache restore at all; we could have just removed this `.next/cache` restore and it would still work the same. Does that make sense? 🙂